tests/test_store.py

```diff
@@ -582,6 +582,7 @@ def test_createTable(
         # assert table exists
```

Comment:
one test failed: ` tests/test_store.py::TestSynapseStorage::test_get_file_entityIds[True] - ValueError: columns overlap but no suffix specified: Index(['entityId'], dtype='object')`

Reply:
For this test, you need to go into your test manifest CSV and delete the ID and entityID columns, similar to the other PR